shared/api/image.go

```diff
@@ -41,6 +41,7 @@ type ImagePut struct {
 	AutoUpdate bool              `json:"auto_update" yaml:"auto_update"`
 	Properties map[string]string `json:"properties" yaml:"properties"`
 	Public     bool              `json:"public" yaml:"public"`
+	ExpiresAt  time.Time         `json:"expires_at" yaml:"expires_at"`
```

Comment:
This field should be put under a `// API extension: images_expiry` comment.

There are a number of similar cases in that file you can look at for the exact formatting.